web/src/main/java/org/cbioportal/web/util/ClinicalDataEnrichmentUtil.java

```diff
@@ -240,6 +239,15 @@
                 sampleIds, attributeIds);
 
         return clinicalDataCountItems.stream()
+                // Exclude NA category
+                .map(clinicalDataCountItem -> {
+                    List<ClinicalDataCount> filteredClinicalDataCount = clinicalDataCountItem.getCounts()
+                            .stream()
+                            .filter(clinicalDataCount -> !clinicalDataCount.getValue().equals("NA"))
```

Comment:
I think that its better to use a constant for "NA" but if this is already how its being done everywhere then okay

Reply:
that's how its done everywhere